.circleci/config.yml

```diff
@@ -0,0 +1,46 @@
+version: 2
+jobs:
+  linting_check:
+    docker:
+      - image: circleci/node:8.9
+    steps:
+      - checkout
+      - restore_cache:
+          key: npm-cache-{{ checksum "package.json" }}    
+      - run:
+          name: Setup lint environment
+          command: npm install
+      - run:
+          name: Lint Test
+          command: npm run lint
+      - save_cache:
+          key: npm-cache-{{ checksum "package.json" }}
+          paths:
+            - ./node_modules
+    
+  publish-package:
```

Comment:
Shouldn't the CircleCi config contain the ```npm publish``` script to publish the new patch version

Reply:
We are going to leave it for now. We are still figuring out this part with permissions etc. I am not 100% comfortable yet with auto publish

Reply:
Makes sense, was just checking 👍